### DISCUSSION TITLE
Call integration for other apps

### DISCUSSION THREAD

**Discussion by Vasmares** [BimmerGestalt/AAIdrive#622] [Ideas]
Hi,

have you ever thought about the integration of other apps to manage calls ?
For example:

I would like to call someone via WhatsApp, Telegram, Signal, Teams.
As of now, I would have to open the app, find the chat and start the call (while driving)
Maybe there is an option to do this via the iDrive interface ?

After the call is established, the car recognizes this as a proper call and switches to the default iDrive call screen.
But starting the call is difficult.

**Comment by hufman** [comment]
I don't know of a good generic way to trigger phone calls in other apps, or whether it would work while the phone screen is locked. I encountered the same obstacle about trying to send new messages to contacts.
